[BUG]ABIFunctionNotFound

## Describe the bug

The following error occurred in our environment.

```
[2022-04-22 09:30:19 +0000] [INDEXER-Position-Bond] [38] [ERROR] ("The function 'pendingTransfer' was not found in this contract's abi. ", 'Are you sure you provided the correct contract abi?')
--
Traceback (most recent call last):
File "batch/indexer_position_bond.py", line 159, in __sync_issuer
balance, pending_transfer = self.__get_account_balance_token(token, issuer_address)
File "batch/indexer_position_bond.py", line 708, in __get_account_balance_token
pending_transfer = ContractUtils.call_function(
File "/app/ibet-Prime/batch/../app/utils/contract_utils.py", line 150, in call_function
_function = getattr(contract.functions, function_name)
File "/home/apl/.pyenv/versions/3.8.12/lib/python3.8/site-packages/web3/contract.py", line 193, in __getattr__
raise ABIFunctionNotFound(
web3.exceptions.ABIFunctionNotFound: ("The function 'pendingTransfer' was not found in this contract's abi. ", 'Are you sure you provided the correct contract abi?')
```

## Expected behavior

We have made the following fixes, but they were not sufficient.

https://github.com/BoostryJP/ibet-Prime/pull/281

## Versions to be fixed
 
- [x] v22.3
- [x]  v22.6

--- a/app/utils/contract_utils.py
+++ b/app/utils/contract_utils.py
@@ -147,9 +147,8 @@ class ContractUtils:
         :param default_returns: Default return when web3 exceptions are raised
         :return: Return from function or default return
         """
-        _function = getattr(contract.functions, function_name)
-
         try:
+            _function = getattr(contract.functions, function_name)
             result = _function(*args).call()
         except (BadFunctionCallOutput, ABIFunctionNotFound) as web3_exception:
             if default_returns is not None:

